### ISSUE TITLE
The main HTML file should not be named `index.html`

### ISSUE THREAD

**Issue by avh4** [elmbridge/elmoji-translator#2]
... because running `elm-make Main.elm` silently overwrites `index.html`.  Some student is going to accidentally do that!

**Comment by raorao** [comment]
should be resolved alongside https://github.com/elmbridge/elmoji-translator/issues/3

**Comment by avh4** [comment]
No longer relevant after #3
